src/Microsoft.Azure.SignalR/HubHost/ServiceLifetimeManagerBase.cs

```diff
@@ -43,6 +43,10 @@ public override Task SendAllAsync(string methodName, object[] args, Cancellation
                 throw new ArgumentException(NullOrEmptyStringErrorMessage, nameof(methodName));
             }
 
+            //return ServiceConnectionContainer.WriteAsync(
```

Comment:
>ServiceConnectionContainer [](start = 21, length = 26)

what if it is a mulitple endpoint container? That one endpoint enables message log while another not?

Reply:
ep == endpoint.
                             epC
                               ^
                               |
Consider epA -> server -> epA
                               | 
                               v
                             epB
And epA and epC enables message log while epB not.
Then message M comes from epA will has message log in server. epA and epC also have message log since they enable.
Due to epB disables message log, so no message log will be generated. If epB wants to have message log, it should enable it. The M stops having message log when the target service disable message log. If the cx disable message log in some ep, we don't want to impact it, unless cx enable manually enable it.